redis/scripts/setup-drupal-settings.sh

```diff
@@ -4,11 +4,11 @@ set -e
 
 if [[ $DDEV_PROJECT_TYPE != drupal* ]] || [[ $DDEV_PROJECT_TYPE =~ ^drupal(6|7)$ ]] ;
 then
-  exit 0
+  exit 1
 fi
 
 if ( ddev debug configyaml 2>/dev/null | grep 'disable_settings_management:\s*true' >/dev/null 2>&1 ) ; then
-  exit 0
+  exit 2
```

Comment:
And why should this be promoted to an error?

Reply:
I agree, this should remain the same, because it's Drupal here.